packages/http-client-java/generator/http-client-generator-test/src/test/java/org/utils/ArmUtils.java

```diff
@@ -39,7 +41,11 @@ public static HttpPipeline createTestHttpPipeline() {
         policies.add(new AddDatePolicy());
         // no ArmChallengeAuthenticationPolicy
         policies.add(new HttpLoggingPolicy(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BODY_AND_HEADERS)));
-        return new HttpPipelineBuilder().policies(policies.toArray(new HttpPipelinePolicy[0])).build();
+        return new HttpPipelineBuilder()
+            .httpClient(new NettyAsyncHttpClientBuilder(
```

Comment:
Personally, I'd think we should fail this test, till core can handle 16k.

Otherwise, we just give confusing info in the dashboard.
We can do this, but user likely cannot (main reason is there is lots of timeout setting that cannot be set here).

Reply:
Agree. Disabled.